CotEditor/tr.lproj/AppearancePane.strings

```diff
@@ -70,7 +70,7 @@
 "csp-Hi-hb1.title" = "Girintileme kılavuzlarını göster";
 
 /* Class = "NSTextFieldCell"; title = "Appearance:"; ObjectID = "6KG-3a-ZuU"; */
-"6KG-3a-ZuU.title" = "Görünüş:";
+"6KG-3a-ZuU.title" = "Görüntü:";
```

Comment:
I thought it's better because macOS uses it for the same purpose. What do you think?

<img width="836" alt="Screenshot 2022-01-31 at 18 15 44" src="https://user-images.githubusercontent.com/1165044/151767575-1f571b5b-9447-4b4c-86a0-7ca8248ee793.png">

You can also check on this website how Apple localize terms in macOS:
https://applelocalization.com/macos